ansible/roles/update_master/tasks/main.yml

```diff
@@ -1,70 +1,70 @@
 ---
```

Comment:
I'm using the update_master role to update the masters for blue/green upgrades in INT. So we should be able to use it for both in-place and blue/green upgrades, since the master portion of the process is the same. I can go ahead and change the name.